src/main/java/org/openelisglobal/qaevent/controller/rest/ReportNonConformEventsRestController.java

```diff
@@ -33,7 +33,7 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
-public class ReportNonConformEventsRestController {
+public class ReportNonConformEventsRestController extends ControllerUtills {
```

Comment:
This is a Rest Controller oler so it  should  extend  the` BaseRestController` instead